### ISSUE TITLE
bedrock: cant enchant fishing rod

### ISSUE THREAD

**Issue by TizoCraft** [GeyserMC/Geyser#2302]
### Describe the bug

so a player reported a issue that he cant enchant his fishing rod. he has enought lvels and lapis but no enchants showed up.

### To Reproduce

try enchanting fishing rod as bedrock player on a java server

### Expected behaviour

e expected that enchants show up

### Screenshots / Videos

https://media.discordapp.net/attachments/814765620226424832/855943092065927168/Screenshot_20210620-004704.jpg?width=1352&height=676

### Server Version and Plugins

purpur-1171-1.16.5

### Geyser Dump

https://dump.geysermc.org/YnTSFjHhHYr5Q5SskbCMO9IA67lZ1HJ9

### Geyser Version

This server is running Geyser version 1.4.0-SNAPSHOT (git-master-cc2d26d) (Java: 1.17, Bedrock: 1.17.0) [05:34:49 INFO]: Checking version, please wait... [05:34:50 INFO]: You are 12 versions behind; download the latest build from https://ci.geysermc.org/

### Minecraft: Bedrock Edition Version

java server is on 1.16.5 i belive bedrock is on the newest 1.17

### Additional Context

im not reachable on github. my discord is Tizo#5051 so if u need more info just add me

**Comment by Camotoy** [comment]
I'm unable to replicate.

**Comment by kystra** [comment]
Hi! So I'm unsure of how to replicate this exactly, but I did manage to encounter this bug twice the other day, and I think I know what's causing it. I enchanted my pickaxe (combined with anvil some books and the tool) maybe once or twice, then went to combine another book. It stated the levels required were 14, 15 with a name change. Tried to combine, it visually appeared to dupe before the glitch went away. I wasn't able to combine them. I logged in with Java some time later, levels required there were 15, 16 with a name change. I'm guessing there's some misreporting going on between levels / penalties between the two, and that's potentially what's causing things? It definitely doesn't happen on the lower enchants, but myself and a few Bedrock players on my server are experiencing this issue. I don't have logs at this time, but will grab them next time I encounter this issue.
